<ui version="4.0" >
 <class>PatternsDlg</class>
 <widget class="QDialog" name="PatternsDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Patterns</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTextEdit" name="m_pTextM" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabChangesFocus" >
      <bool>true</bool>
     </property>
     <property name="acceptRichText" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QTextBrowser" name="m_infoM" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>9</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QWidget" native="1" name="widget" >
     <layout class="QHBoxLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="m_pAddPredefB" >
        <property name="text" >
         <string>Add predefined patterns</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="m_pOkB" >
        <property name="text" >
         <string>O&amp;K</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pCancelB" >
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
